components/raftstore/src/store/peer.rs

```diff
@@ -910,6 +912,13 @@ where
     ///   target peer.
     /// - all read requests must be rejected.
     pub pending_remove: bool,
+    /// Currently it's used to indicate whether the witness -> non-witess
+    /// convertion operation is complete. The meaning of completion is that
+    /// this peer must contain the applied data, then PD can consider that
+    /// the conversion operation is complete, and can continue to schedule
+    /// other operators to prevent the existence of multiple witnesses in
+    /// the same time period.
+    pub initialized: bool,
```

Comment:
Why not name it `miss_data`?

Reply:
Well, I wanted to be more general, but agree that miss data is more precise.